src/SIL.XForge/Services/UserService.cs

```diff
@@ -69,9 +69,12 @@ await using (IConnection conn = await _realtimeService.ConnectAsync(curUserId))
             );
             await userDoc.SubmitJson0OpAsync(op =>
             {
+                string picture =
```

Comment:
I found this block a little confusing on first glance (I had to double take to figure out what it was doing). One way you could make this more readable is to simplify this line as follows:
```csharp
string picture = (string)userProfile["user_metadata"]?["picture"];
```